src/ro/onlinelibrary/library/ServiceOnlineLibrary.java

```diff
@@ -0,0 +1,83 @@
+package ro.onlinelibrary.library;

```

Comment:
create a separate package for services - name it service and group there the interface and the service implementation